verilog/tools/ls/autoexpand.cc

```diff
@@ -18,10 +18,14 @@
 #include <algorithm>
 #include <regex>
 
+#include "absl/container/flat_hash_map.h"
 #include "common/text/text_structure.h"
+#include "verilog/CST/declaration.h"
```

Comment:
Did you add all the necessary CST dependencies in the BUILD file ? It might be that they link by accident, but we should be explicit.

Reply:
Missed it, I added them now